config/migrations/2024/20241126144504-rename-merged-municipalities-and-ocmws.sparql

```diff
@@ -0,0 +1,25 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?organisation skos:prefLabel ?oldName.
+  }
+} INSERT {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?organisation skos:prefLabel ?newName.
+  }
```

Comment:
For the `mock-login` part, we can add `?newName` as the new family name for these mock persons:
```sparql
GRAPH ?g {
  ?person foaf:familyName ?newName .
}
```